CONTRIBUTING.md

```diff
@@ -15,3 +17,29 @@
 * Be descriptive about the problem and reason about the proposed solution
 * Include release notes describing the potential impact of the change in the pull request
 * Make sure there has been at least one approval from Shopify before merging
+
+## Release
+
+After your changes update [CHANGELOG.md](./CHANGELOG.md) with summary of all user facing changes.
```

Comment:
There is no need to change the CHANGELOG and create a release in the releases page. This will just make the CHANGELOG be out of sync with the releases page since it can't be changed inside the same release while the releases page can.